generators/app/package.json

```diff
@@ -2,18 +2,22 @@
   "name": "polymer-starter-kit-custom-build",
   "version": "0.0.0",
   "description": "A starting point for Polymer 1.0 custom build apps",
-  "license": "http://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
```

Comment:
@robdodson hum... but we have other in the [bower.json#L6](https://github.com/PolymerElements/polymer-starter-kit/blob/master/bower.json#L6). We need to fix that. 😅